README.md

```diff
@@ -101,19 +101,27 @@ npm install --save pouchdb-hoodie-store
 Attach this plugin to the `PouchDB` object:
 
 ```js
-var PouchDB = require('pouchdb')
-PouchDB.plugin(require('pouchdb-hoodie-store'))
+var Store = require('pouchdb-hoodie-store')
 ```
 
 #### In the browser
 
-Include this plugin after `pouchdb.js` in your HTML page:
+Include this plugin in your HTML page:
 
 ```html
-<script src="node_modules/pouchdb/dist/pouchdb.js"></script>
 <script src="node_modules/pouchdb-hoodie-store/dist/pouchdb-hoodie-store.js"></script>
 ```
 
+Since `pouchdb.js` is bundled by default into the plugin, there's no need to load separately.  
+If you want to load your own PouchDB, just add before loading the plugin, now it'll use your `pouchdb.js` 
+  
+```html
+<script src="node_modules/pouchdb/dist/pouchdb.js"></script>
+```
+
+Additionally we will dist a "without PouchDB" version `pouchdb-hoodie-store-without-pouchdb.js` which you  
```

Comment:
Might be best to say "distribute" rather than abbreviate to "dist". That's a personal preference for readability though.